jobs/registry/templates/bin/ctl

```diff
@@ -11,6 +11,15 @@ export LANG=en_US.UTF-8
 case $1 in
   start)
     pid_guard $PIDFILE $JOB_NAME
+    <%- if_p('docker.registry.http_proxy') do |v| %>
+      export http_proxy=<%=p('docker.registry.http_proxy') %>
```

Comment:
You don't need the `if_p(...) { |v| ... }` block construct if you're going to refer to the property by name via `p(...)`; you can either just use `v` in the body of the block, or check if `p(...)` is non-blank.

Reply:
you are 100% right 👍, we will fix that ASAP

Reply:
@jhunt have a look now, pls